submissions/2025-05-01-OpenFF-Sage-2.0.0-Optimization-Training-Dataset-v1.1/README.md

```diff
@@ -0,0 +1,69 @@
+# OpenFF Sage 2.0.0 Training Optimization v1.1
+
+### Description
+
+A quantum chemical (QC) dataset curated to train [OpenFF 2.0.0 Sage](https://github.com/openforcefield/openff-sage) forcefield, with reparametrized Lennard-Jones (LJ) and valence parameters, the latter relevant to this dataset. This QC dataset with the OpenFF default level of theory, B3LYP-D3BJ/DZVP, is used to train Sage parameters. These optimized conformer geometries were used in conjunction with the QC dataset used to train one dimensional torsional profiles. This Generation 2 dataset increases chemical diversity when compared to Generation 1, which are of value to our industry partners. Large molecules (>20 heavy atoms) were also included, including more flexible molecules and a greater degree of conformational variation which provide intramolecular interactions.
+
+This is the complete optimization dataset used for training OpenFF 2.0.0 Sage, consisting of molecules from following datasets:
+
+ - [OpenFF Gen 2 Opt Set 1 Roche](https://github.com/openforcefield/qca-dataset-submission/tree/0e6e6da930118e2a2d6402b93c3e3e93830600cc/submissions/2020-03-20-OpenFF-Gen-2-Optimization-Set-1-Roche)
+ - [OpenFF Gen 2 Opt Set 2 Coverage](https://github.com/openforcefield/qca-dataset-submission/tree/0e6e6da930118e2a2d6402b93c3e3e93830600cc/submissions/2020-03-20-OpenFF-Gen-2-Optimization-Set-2-Coverage)
+ - [OpenFF Gen 2 Opt Set 3 Pfizer Discrepancy](https://github.com/openforcefield/qca-dataset-submission/tree/0e6e6da930118e2a2d6402b93c3e3e93830600cc/submissions/2020-03-20-OpenFF-Gen-2-Optimization-Set-3-Pfizer-Discrepancy)
+ - [OpenFF Gen 2 Opt Set 4 eMolecules Discrepancy](https://github.com/openforcefield/qca-dataset-submission/tree/0e6e6da930118e2a2d6402b93c3e3e93830600cc/submissions/2020-03-20-OpenFF-Gen-2-Optimization-Set-4-eMolecules-Discrepancy)
+ - [OpenFF Gen 2 Opt Set 5 Bayer](https://github.com/openforcefield/qca-dataset-submission/tree/0e6e6da930118e2a2d6402b93c3e3e93830600cc/submissions/2020-03-20-OpenFF-Gen-2-Optimization-Set-5-Bayer)
+
+The following filters were applied to those datasets:
+
+ - `RecordStatusFilter(status=RecordStatusEnum.complete)`
+ - `ConnectivityFilter(tolerance=1.2)`
+ - `UndefinedStereoFilter()`
+ - `ElementFilter(allowed_elements=["H", "C", "N", "O", "S", "P", "F", "Cl", "Br", "I"])`
+ - `ConformerRMSDFilter(max_conformers=10)`
+
+ Further information can be found in the curation scripts for the linked repositories.
+
+### General Information
+
+- Date: 2025-05-01
+- Class: OpenFF Optimization Dataset
+- Purpose: Complete set of training data for OpenFF 2.0.0 Sage
+- Dataset Type: optimization
+- Name: OpenFF Sage 2.0.0 Training Optimization Dataset v1.1
```

Comment:
```suggestion
- Name: OpenFF Sage 2.0.0 Optimization Training Dataset v1.1
```
Could you please match the name to the directory? The Training and Optimization are still swapped around